Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/KotlinCodeModel.kt

```diff
@@ -320,7 +320,7 @@ abstract class KotlinFile(
                 // TODO: using original global name should be preferred to importing the clashed name.
             } else {
                 declaredProperties.add(name)
-                name.asSimpleName()
+                name
```

Comment:
Have you ever consider different types for MangledName and non-mangled? Even if they have the same text representation (for most of cases I guess) they still have different usage areas.

Reply:
Yeah, it's may be a good idea to use `inline class` for such thing.